components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java

```diff
@@ -68,43 +75,58 @@ public Set<String> getMethods() {
     }
 
     @Override
-    public boolean invoke(MessageContext msgCtx) {
-        buildMessage(msgCtx);
+    public boolean invoke(MessageContext messageContext) {
+        buildMessage(messageContext);
```

Comment:
Shall we keep the same variable name to preserve edit history. There is no functionality change right?